### PR TITLE
Selecting Files

### PR DESCRIPTION
**Pull request by qvalentin**
you will be able to select file from different folders at the same time. This way you can easily select what you want to delete/download.

Cypress tests are broken because cors is not working on the server.

### CODE REVIEW

File: webapp_frontend/src/components/pages/filesytem/FileList.tsx [hunk 1]
```diff
@@ -48,6 +69,28 @@ export default function FileList(props: Props): ReactElement {
...
+    }, [folders, files, currentFolderSelected])
+
+    const handleSelectAllChanged = () => {
+
+        if (allAreSelected) {
+            files?.forEach((e: FsEntity) => props.removeFromSelected(e));
+            folders?.forEach((e: FsEntity) => props.removeFromSelected(e));
+        } else {
+            files?.forEach((e: FsEntity) => props.addToSelected(e));
+            folders?.forEach((e: FsEntity) => props.addToSelected(e));
```
Comment (qvalentin) [inline webapp_frontend/src/components/pages/filesytem/FileList.tsx]: this could cause doubled entries

**Comment by open-schnick** [comment]
gonna review this tomorrow

**Comment by open-schnick** [comment]
Looks good to me. Is the feature the fact that you can select the files with a checkbox?.  
One more thing tho. Can we implement a fixed placeholder for the "x files selected"? I am kinda annoyed by the fact that the whole page shifts downwards, when first clicking. Also a good implementation could be a toast at the bottom of the page (like on dropbox)

File: webapp_frontend/src/components/pages/filesytem/FileList.tsx [hunk 2]
```diff
@@ -1,107 +1,169 @@
...
+  //     console.log(folders)
+  //     let foldersa = folders ? [...folders] : []
+  //     let filesa = files ? [...files] : []
+  //     let sortedFoldersa = sortObjectsInArrayByProperty(foldersa, "name")
+  //     let sortedFilesa = sortObjectsInArrayByProperty(filesa, "name")
+  //     console.log(sortedFoldersa)
+  //     console.log("---------")
+  //     console.log(filesa)
+  //     console.log()
+  // console.log("--------------------------------------------------------------------------------------")
```
Comment (open-schnick) [inline webapp_frontend/src/components/pages/filesytem/FileList.tsx]: production ready?

**Review by open-schnick — CHANGES_REQUESTED** [review]
code does look fine to me.

File: webapp_frontend/src/components/pages/filesytem/FileList.tsx (hunk 2 above)
Comment (qvalentin) [reply webapp_frontend/src/components/pages/filesytem/FileList.tsx]: @gimleux fault :)

**Review by open-schnick — APPROVED** [review]

**Comment by open-schnick** [comment]
merge it if you want

**Pull request merged**